base/utils/R/utils.R

```diff
@@ -512,7 +512,7 @@ temp.settings <- function(settings.txt) {
 ##' @author David LeBauer
 tryl <- function(FUN) {
   out <- tryCatch(FUN, error = function(e) e)
-  ans <- !any(class(out) == "error")
+  ans <- !any(is.error(out))
```

Comment:
Similarly to above, there is no such function `is.error`. Since we're doing `tidyverse` anyway throughout our code, I'd probably recommend scrapping this function altogether and using some of the `purrr` error-handling adverbs (e.g. `purrr::possibly`).

Reply:
@ashiklom if we use `purrr` here than we won't get a boolean type return & basically we need to do the type conversion. So I rather prefered using `inherits` here. Please let me know if you want any changes.

Reply:
I think @ashiklom's talking about replacing the whole`tryl` function, not about using `purrr::possibly` on this line of it. You're correct to use `inherits` here for this PR, whether or not `tryl` gets removed in a future update.